src/PEImage.cpp

```diff
@@ -331,7 +331,7 @@ bool PEImage::initDbgPtr(bool initDbgDir)
 
 	symtable = (char*)(sec + nsec);
 	nsym = dbg->ExportedNamesSize;
-	strtable = symtable + nsym * IMAGE_SIZEOF_SYMBOL;
+	strtable = symtable + nsym;

```

Comment:
`nsym` is probably wrong to begin with, should be `dbg->ExportedNamesSize/IMAGE_SIZEOF_SYMBOL` and `symtable + dbg->ExportedNamesSize` used here.

Reply:
ExportedNames block consist of several null-terminated symbol names with arbitrary length written one after another.
I can send you an example of such file if you need.

Reply:
Oops. It seems I've committed wrong version of the file.